specifyweb/frontend/js_src/lib/localization/locality.ts

```diff
@@ -292,4 +292,10 @@ export const localityText = createDictionary({
     'uk-ua': 'DD MM SS.SS N/S/E/W (32 45 42.84 N)',
     'de-ch': 'DD MM SS.SS N/S/O/W (32 45 42.84 N)',
   },
+  validLatitude: {
```

Comment:
(optional) would it make sense to reuse one of these strings?
https://github.com/specify/specify7/blob/4e5722aabb4ae3e8afd2ee487f4bea68eaff83d1/specifyweb/frontend/js_src/lib/localization/backEnd.ts#L178-L211

Reply:
I don't mind the new string personally. I think it's more concise as `Actual: (value)` is not really needed on the frontend